src/api.rs

```diff
@@ -458,7 +462,7 @@ pub struct Config {
   pub threads: usize
 }
 
-const MAX_USABLE_THREADS: usize = 4;
+const MAX_USABLE_THREADS: usize = 8;
```

Comment:
The reason for this is because the intra mode rayon usage is really too fine grained to scale well, while for tile threading we don't need this limit. Instead of bumping this number we should probably remove or limit the intra mode parallelism.

Reply:
OK, so I remove this commit for now?

Indeed, the solution is probably to not submit too fine grained tasks to rayon and totally remove this limit.

Reply:
Given discussion on irc, just drop the pmv par_iter and remove the MAX_USABLE_THREADS threshold entirely.